### DISCUSSION TITLE
HP Probook 6470b - Ventura without audio and wifi

### DISCUSSION THREAD

**Discussion by spn0381** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#180] [General]
hello everyone. I have just installed Ventura on HP Probook 6470b  (bios B74) and after:

I've PostInstalled Everything as described in the video and then rebooted.
The video work good only after post Install OCPL

But Wifi and Audio doesn't works... any idea about them? (On Big Sur, the previous installation, they works good...

thank you in advance for your help...

**Comment by spn0381** [comment]
solved... after Post Install OCLP and installing again 2 times the Chris post install package ... worked

**Reply by morganinh4** [reply]
@spn0381 hello, I have the same laptop but I'm on big sur... Can you tell me what your wifi card is? The one on my hp probook 6470b never worked on all mac.

Is the bluetooth working normally on the ventura? Can you please test if the game "league of legends" installs? last time I tested ventura, league of legends not open after install, it always crashed... Thanks.

**Reply by spn0381** [reply]
![IMG_3422](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/17476534/26542140-fb4d-4078-ab3d-67b51a151112)
![IMG_3421](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/17476534/549501bf-97c7-4ca0-abac-059eec98cfa0)
The Bluetooth works perfectly, my ear pods pro are connected and fully working.

my Wi-Fi card is a Broadcom BCM43xx, and worked also in big sur 

league of legend crash. It seems that after downloading 250 mb more or less, open an agent and then crashed …

now I’m updating to 13.5
![IMG_3422](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/17476534/26542140-fb4d-4078-ab3d-67b51a151112)

**Reply by morganinh4** [reply]
@spn0381 thanks for your answers. I had the same issue on venturra 13.2. after installed it doesn't open anymore. But it works perfectly in big sur and monterey... Thanks

**Reply by spn0381** [reply]
I don’t think that’s can be an hackintosh problem …

**Reply by morganinh4** [reply]
@spn0381 can you test Chris solution in league of legends?

**Comment by chris1111** [comment]
People using boot args `amfi=0x80`
take a look to this for App Crash
[tccplu](https://github.com/jslegendre/tccplus)

**Reply by morganinh4** [reply]
hello @chris1111 I tested it on ventura 13.5 but it's not working for me... I don't know what the problem is because in big sur and monterey it works fine, check screenshots. 
![Captura de Tela 2023-07-29 às 11 33 46](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/21047444/e406c76e-a95d-45f5-939a-fe356a0a6f25)
![Captura de Tela 2023-07-29 às 11 51 56](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/21047444/e81d6ce1-cb4b-4c99-8156-e216190c6e11)
![Captura de Tela 2023-07-29 às 11 55 22](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/21047444/14b76c8a-7c9a-4954-886c-beb28fc9b8c8)
![Captura de Tela 2023-07-29 às 11 58 33 AM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/21047444/1cc59814-8e4b-4d74-a100-8966815fe6ee)

**Reply by morganinh4** [reply]
Hello @spn0381 @chris1111 

I found the solution and I came back here to share it. It might be useful for those who are having problems with games. I needed to do two things:

For non-AVX2 CPUs, you need to disable f16c sysctl reporting to resolve CoreGraphics.framework crashing.
And if you are using Metal 1 dGPU, disable mediaanalysisd which requires Metal 2 feature set! This is in NVRAM >> boot-args: 'revpatch=16c' (to disable avx2) and 'revblock=media' (to disable metal 2).
After I made these changes everything works!

**Reply by spn0381** [reply]
In normal use: there are enhancements or not ??

**Reply by morganinh4** [reply]
@spn0381 not sure yet, but with revblock=media I get the best FPS in games... So maybe it helps render other apps too.

**Reply by morganinh4** [reply]
@spn0381 hello, I intend to buy a wifi card, I researched about BCM43xx and there are many models. Can you tell me the exact model of yours? Mine is a 6470b only hd4000, works on ventura?

**Reply by spn0381** [reply]
My hp Probook 6470b works perfectly with the package by Chris1111, everything is working ok (only Intel hd 4000).
My wifi card is: BCM943224HMS DW1520

**Comment by chris1111** [comment]
whats not working?

**Reply by morganinh4** [reply]
@chris1111 the game 'league of legends', not open after installation. It has this dark window and then it closes after 2/3 secs
